libkatana_python_native/include/katana/python/ErrorHandling.h

```diff
@@ -47,6 +47,56 @@ PythonChecked(katana::Result<T>&& src) {
   return std::move(src.value());
 }
 
+template <typename Cls, typename Return, typename Error, typename... Args>
```

Comment:
Had to look through these carefully to even see the difference!

Reply:
Yea. For some god forsaken reason, there is no way to abstract over constness of method pointers.